tests/unittest.py

```diff
@@ -241,7 +241,7 @@ def setUp(self):
         # create a site to wrap the resource.
         self.site = SynapseSite(
             logger_name="synapse.access.http.fake",
-            site_tag="test",
+            site_tag=self.hs.config.server.server_name,
```

Comment:
This is just a helpful thing to make logging in tests use the server name as the tag. Workers will use `<server_name>-<instance_name>`